packages/fx-core/src/plugins/resource/bot/dotnet/constants.ts

```diff
@@ -11,4 +11,15 @@ export class PathInfo {
     "bicep"
   );
   public static readonly ProvisionModuleTemplateFileName = "botServiceProvision.template.bicep";
+  static readonly appSettingDevelopment = "appsettings.Development.json";
+}
+
+export class DependentPluginInfo {
```

Comment:
Since the logic exist in Frontend plugin before, so it use dependentPluginInfo. But as we move the logic to Bot plugin itself, do we still need this?

Reply:
This class is removed.